bundles/AdminBundle/Resources/public/js/pimcore/helpers/quantityValue.js

```diff
@@ -22,7 +22,11 @@ pimcore.helpers.quantityValue.initUnitStore = function(callback, filters) {
         var newListener = function () {
             pimcore.helpers.quantityValue.storeLoaded = true;
             pimcore.helpers.quantityValue.storeLoading = false;
-            pimcore.helpers.quantityValue.getData(callback, filters);
+            try {
+                pimcore.helpers.quantityValue.getData(callback, filters);
```

Comment:
wouldn't it be better to filter it out in getData instead of letting the entire call fail ?

Reply:
Please have look at my latest commits. I didn't exactly know what you ment with "filter it".

Reply:
My impression was that getData allows to pass in a list of IDs